app/src/main/java/com/example/triptracker/view/profile/UserProfileFollowers.kt

```diff
@@ -30,92 +33,102 @@ import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.example.triptracker.R
+import com.example.triptracker.model.profile.Relationship
 import com.example.triptracker.model.profile.UserProfile
 import com.example.triptracker.view.Navigation
 import com.example.triptracker.view.NavigationBar
 import com.example.triptracker.view.theme.md_theme_light_dark
 import com.example.triptracker.viewmodel.UserProfileViewModel
+import com.example.triptracker.viewmodel.loggedUser
 
 /** This composable function displays the user's following list. */
 @Composable
 fun UserProfileFollowers(
     navigation: Navigation,
-    userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),
-    userProfile: UserProfile
+    viewModel: UserProfileViewModel = UserProfileViewModel(),
```

Comment:
Good change! Maybe keeping the parameter name as userProfileViewModel is a bit clearer than viewModel when using the UserProfileFollowers function.